conda-forge.yml

```
@@ -4,12 +4,14 @@ os_version:
   linux_aarch64: cos7
   linux_ppc64le: cos7
 provider:
-  linux_ppc64le: azure
-  linux_aarch64: azure
+  linux_aarch64: default
+  linux_ppc64le: default
```

Sorry meant to keep the original settings (before this PR)

```suggestion
  linux_aarch64: azure
  linux_ppc64le: azure
```

---

Ah ok, got it. Thanks John!